packages/react-label/src/components/Label/useLabelStyles.ts

```diff
@@ -7,8 +7,8 @@ import { LabelState } from './Label.types';
 const useStyles = makeStyles({
   root: theme => ({
     fontFamily: theme.global.type.fontFamilies.base,
-    fontSize: theme.global.type.fontSizes.base[300],
     color: theme.alias.color.neutral.neutralForeground1,
+    boxSizing: 'content-box',
```

Comment:
Is there any reason to do `content-box` specifically? I've gotten used to global `box-sizing: border-box` resets, and this seems like it might be a bit unexpected.

Reply:
there isn't, if preferred I can remove it.